### ISSUE TITLE
Vitess fuzzers are broken

### ISSUE THREAD

**Issue by dbussink** [cncf/cncf-fuzzing#533]
Currently the Vitess fuzzers are broken and have been for a while. There have been refactors on the Vitess end without realizing that they would affect things here.

I tried https://github.com/vitessio/vitess/pull/17380, but it looks like there's copied set in this repository. I'm not sure how to test / fix that though.

@AdamKorcz Would you have any input / way to fix this here? Looks like you did most of the Vitess additions here?

**Comment by AdamKorcz** [comment]
Yes, I have been handling the Vitess fuzzers over several years. Ideally, they should live in the Vitess repository, such that changes to the Vitess source don't break the fuzz build. I will be happy to clean up all the fuzzers and migrate them upstream if you will accept the PRs.

**Comment by dbussink** [comment]
> Yes, I have been handling the Vitess fuzzers over several years. Ideally, they should live in the Vitess repository, such that changes to the Vitess source don't break the fuzz build. I will be happy to clean up all the fuzzers and migrate them upstream if you will accept the PRs.

That sounds like a good idea then to keep them in Vitess so it's easier to maintain and avoid regressions here.
